services/auth-svc/.env.example

```diff
@@ -13,10 +13,6 @@ PORT=8080
 # The port on which the http server will listen on (default: 8081)
 HTTP_PORT=8081
 
-# port on which the dapr sidecar listens on
-# in prod this is managed by the dapr-sidecar-injector
-DAPR_HTTP_PORT=
-
 #
 # Keycloak / OIDC config
```

Comment:
I think github chose to only show the code above the line I commented on. I meant everything below it though. The HTTP_PORT can stay

Reply:
We will clean this up when we are about to remove Keycloak entirely.